tailwind/styles.css

```diff
@@ -36,7 +36,13 @@ a { @apply text-red-600 underline; }
 
 ol, ul { padding-left: 1em; }
 
-li { @apply list-outside list-disc; }
+ol {
+  list-style: decimal;
+}
+
+ul {
+  list-style: disc;
+}
```

Comment:
Since these two are already the default, it should be possible to just remove them altogether.

Reply:
Thanks @ezio-melotti for the comment.  @cstnaya what do you think about making a new PR for it?

Reply:
Within using `@tailwind base;`, lists will be set to unstyled, which made `ul` & `ol` markers are all invisible.
I want to overwrite this setting, and that's why I wrote down those lines.

Is it a better way if I replace the codes with lines below?
```
@layer base {
  ol, ul {
    list-style: revert;
  }
}
```

Reply:
> Is it a better way if I replace the codes with lines below?

I am not familiar with tailwind enough to comment on that.  @chestercheng what do you think?